python/ray/_private/import_thread.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, worker, mode, threads_stopped):
         self.gcs_client = worker.gcs_client
         self.threads_stopped = threads_stopped
         self.imported_collision_identifiers = defaultdict(int)
+        # Keep track of the number of imports that we've imported.
+        self.num_imported = 0
```

Comment:
We filter out function keys not from our job right (this is the change @jjyao introduced) So could this cause issues where num_imported is much less than the number of functions?

Reply:
I think it won't impact the num_imported. It happens in _process key, this field actually means the key scanned.

Reply:
Wont we still trigger a KV operation (lindex) even if skipping the key? That could cause a lot of extra load on the GCS. Maybe we should fetch the entire list in one redis operation.

Reply:
Yes, but it's the same as before. Previously, it's fetching data from redis, and then skip the key.
I think it's ok here. If it's a bottleneck we can optimize it later. (per-job maybe?)